cloud: update doc for left navigation (#11568)
pingcap/docs: tidb-cloud/create-tidb-cluster.md

To create a Serverless Tier cluster, take the following steps:

1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.

2. Click **Create Cluster**.

> 3. On the **Create Cluster** page, update the default cluster name if necessary.

Reviewer: ```suggestion
3. On the **Create Cluster** page, select **Serverless Tier**, and update the default cluster name if necessary.
```

4. Note that the cloud provider of Serverless Tier is AWS, and then select the region where you want to create your cluster.

5. Click **Create**.

    The cluster creation process starts and your TiDB Cloud cluster will be created in approximately 30 seconds.

> 6. During the creation process, perform security settings for your cluster:

Reviewer: ```suggestion
6. After the creation is completed, perform security settings for your cluster:
```
